signer/tests/lib_test.rs

```diff
@@ -290,7 +291,7 @@ fn sign_bls_transaction() {
     let bls_key =
         PrivateKey::try_from(test_value["bls_private_key"].as_str().unwrap().to_string()).unwrap();
 
-    println!("{}", bls_address.to_string());
+    dbg!(bls_address.to_string());
```

Comment:
Nice!

Reply:
I wasn't sure whether to add it or not, but it just improved the test output readability, and it didn't look like it was being processed by CI